sass/components/_variables.scss

```diff
@@ -35,14 +35,25 @@
 // ==========================================================================
 
 $primary-color: color("materialize-red", "lighten-2") !default;
-$primary-color-light: lighten($primary-color, 15%) !default;
-$primary-color-dark: darken($primary-color, 15%) !default;
+$primary-color-font-emphasize-color: $primary-color !default;
+$primary-color-font-medium-color: rgba($primary-color, 0.7) !default;
+$primary-color-font-disabled-color: rgba($primary-color, 0.4) !default;
+$primary-color-hover-opaque: rgba($primary-color, 0.06) !default;
+$primary-color-focus-opaque: rgba($primary-color, 0.18) !default;
```

Comment:
it will be nice to remove the 0 value on opacity  to have something more consistent